src/OrbitGrpcProtos/capture.proto

```diff
@@ -56,7 +58,7 @@ message SchedulingSlice {
 message FunctionCall {
   int32 pid = 1;
   int32 tid = 2;
-  uint64 absolute_address = 3;
+  uint64 function_id = 3;
```

Comment:
So in the end the address is not sent back? Are we sure we don't need it?

Reply:
we never did send it back, we only sent whatever client sent us in instrumented function list. this did not change - I only renamed it to id.

(and yes - we do not need it in FunctionCall after this change)

Reply:
True.